app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/FavoritesListAdapter.java

```diff
@@ -34,29 +37,28 @@
 public final class FavoritesListAdapter extends Adapter {
 
     // stores all routes for favorite at the position represented by the key
-    private HashMap<Integer, ArrayList<Route>> mAllRoutesToFavorites = new HashMap<Integer, ArrayList<Route>>();
+    private HashMap<Integer, SectionedRoutes>
```

Comment:
you can do `private HashMap<Integer, SectionedRoutes> m...= new HashMap<>();`, and Java will infer the types!